plugins/quetz_sql_authenticator/tests/test_quetz_sql_authenticator.py

```diff
@@ -0,0 +1,105 @@
+from contextlib import contextmanager
+from unittest import mock
+
+from click.testing import CliRunner
+from quetz_sql_authenticator.cli import _create, _delete, _reset, calculate_hash
+from quetz_sql_authenticator.db_models import Credentials
+
+
+# Test invalid login
+def test_invalid_login(client):
+    response = client.post(
+        "/auth/sql/authorize",
+        data={"username": "testuser", "password": "testpassword"},
+    )
+    assert response.status_code == 200
```

Comment:
is it sensible to return a status code 200 here?

Reply:
401 is probably better 👌🏼

Reply:
But this would mean changing code in quetz core, which a bunch of other stuff probably relies upon. I'll see whether the tests still pass if I change this.

Reply:
Tests seem to be [passing](https://github.com/janjagusch/quetz/actions/runs/3280901774).